recipe/meta.yaml

```
@@ -85,7 +85,7 @@ outputs:
 
 about:
   doc_url: https://evalml.featurelabs.com/en/stable/
```

- Better to let Read the Docs redirect 
```suggestion
  doc_url: https://evalml.featurelabs.com
```